Re-Target: Missing commas in bib(la)tex.

Xaldew opened the pull request:
Re-targeting of pull request as mentioned in #1091, and should thus close issue #1092.

hollasch commented:
Sorry, you need to sync your fork, then make your changes to `dev-major` (that's our current development branch), and then issue a PR to merge into _our_ `dev-major` branch. Also, please add a line to CHANGELOG.md, and update `books/ackowledgments.md` with your name. Thank you!

Xaldew commented:
Ah, you're right, I tried to finish stuff up way too quickly and way too late at night so I didn't notice the obviously wrong change history. Anyways, I've changed the source branch, added changelog entry and added my name to the `books/acknowledgements.md.html` (although I personally think it is a a bit too small for that, but still). So everything seems to be in better order now, at least as far as I can tell, but do shout if something is missing

Again, sorry for the inconvenience!

hollasch commented:
Thank you Gustaf!
